velox/exec/RowsStreamingWindowBuild.cpp

```diff
@@ -101,13 +122,18 @@ void RowsStreamingWindowBuild::noMoreInput() {
 }
 
 std::shared_ptr<WindowPartition> RowsStreamingWindowBuild::nextPartition() {
-  VELOX_CHECK(hasNextPartition());
-  return windowPartitions_[++outputPartition_];
+  VELOX_CHECK(!windowPartitions_.empty());
```

Comment:
Remove this as outputPartition has the same check? Thanks!

Reply:
@xiaoxmeng Yes. Removed.